### DISCUSSION TITLE
Dead-end onboarding workflow for single-user instances

### DISCUSSION THREAD

**Discussion by elsiehupp** [JulianPrieber/littlelink-custom#476] [Feature requests]
> ***Note:*** *This is kind of a duplicate of https://github.com/LinkStackOrg/LinkStack/issues/449*.

When setting up a new LinkStack instance, if you tick the checkbox to use the administrator's profile as the homepage, once you finish the onboarding process, it dumps you onto the profile page with no obvious way to log in.

When I found myself in this situation, I looked at [the LinkStack documentation](https://docs.linkstack.org/), and it didn't explain how to log in, so I just guessed `/admin` and then `/login` before being able to log in. (I assume `/dashboard` would have worked, as well.)

Probably the most user-friendly fix for this would be to add a preference to show or hide a "Log In"/"Dashboard" link on each profile, with the preference enabled by default if the administrator sets their profile as the homepage during initial setup and disabled by default otherwise.

Making this a preference that can be changed after the fact would cover for particular admins not preferring these defaults, but as a heuristic it would help prevent the situation I found myself in, literally having to guess the URL for the dashboard login page.

**Comment by JulianPrieber** [comment]
My idea would be to disable the home page in single user mode by default and maybe display a login button on the links page/chnage the home link in the footer into a login button.

**Reply by elsiehupp** [reply]
I mean that makes sense.

I'm curious, though: how do you categorize between issues and discussions here?

**Reply by JulianPrieber** [reply]
Well, we mainly use discussions for planning and feature requests. Things that lack a clear implementation path, such as unresolved implementation methods, require additional planning, or are feature requests.

If it's something that is an immediate problem, has a clear implementation path, is a user problem, or a bug, it's an issue.

I generally move issues to discussion if we need more user feedback on them or if the issue becomes more permanent or is unclear when we are able to add it in.

It's mostly easier to manage future features with the categories here.
